### PR TITLE
Updated to leverate the latest cloneable readable

### PR DESCRIPTION
**Pull request by mcollina**
Updated for https://github.com/mcollina/cloneable-readable/pull/3

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -111,10 +111,6 @@ File.prototype.clone = function(opt) {
   // Clone our file contents
   var contents;
   if (this.isStream()) {
-    if (typeof this.contents.clone !== 'function') {
```
Comment (phated) [inline index.js]: Should this be removed?  I'm concerned that someone may set `file._contents` to a non-wrapped stream.

Comment (mcollina) [inline index.js]: A couple of notes here:

a. the API compatible way to set `file._contents` is via `file.contents`. Are we supporting direct access to `_contents`? If so, yes, we should keep that check.
b. I've added a `cloneable.isCloneable(stream)` which is stricter.

Comment (phated) [inline index.js]: Fair point, I am always scared that people will use anything exposed (even did a talk about it!)

**Comment by mcollina** [comment]
I agree with you, but that's when I need to do something that is not supported by the API. contents is widely available and well documented.

**Comment by phated** [comment]
Thanks @mcollina!  The tests are working awesome now.

**Pull request merged**